### PR TITLE
Commercial schedules RI changes

### PR DESCRIPTION
**Pull request by palatsangeetha**

### CODE REVIEW

File: commercialschedules-application/src/main/java/org/dcsa/commercialschedules/controller/CommercialSchedulesController.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        .voyageNumber(voyageNumber)
+        .universalVoyageReference(universalVoyageReference)
+        .unLocationCode(unLocationCode)
+        .facilitySMDGCode(facilitySMDGCode)
+        .startDate(startDate)
+        .endDate(endDate)
+        .limit(limit)
+        .build());
+
+    paginator.setPageHeaders(request, response, cursor, result);
```
Comment (douradofrazer) [inline commercialschedules-application/src/main/java/org/dcsa/commercialschedules/controller/CommercialSchedulesController.java:75]: Pagination missing for the below two APIs.

File: commercialschedules-domain/src/main/init.db/initialize-db.sql [hunk 2]
```diff
@@ -3,8 +3,6 @@
 
 \set ON_ERROR_STOP true
 
-CREATE USER dcsa_db_owner WITH PASSWORD '9c072fe8-c59c-11ea-b8d1-7b6577e9f3f5';
-CREATE DATABASE dcsa_template OWNER dcsa_db_owner;
+CREATE DATABASE dcsa_commercialschedules;
```
Comment (douradofrazer) [inline commercialschedules-domain/src/main/init.db/initialize-db.sql:6]: `dcsa_cs` to keep it consistent with how we have been naming so far.

**Review by douradofrazer — COMMENTED** [review]
I also think the packages should be remained as `org/dcsa/cs` to keep it consistent to how we have been doing it so far.

File: commercialschedules-domain/src/main/init.db/initialize-db.sql (hunk 2 above)
Comment (palatsangeetha) [reply commercialschedules-domain/src/main/init.db/initialize-db.sql]: Done the naming changes

File: commercialschedules-application/src/main/java/org/dcsa/commercialschedules/controller/CommercialSchedulesController.java (hunk 1 above)
Comment (palatsangeetha) [reply commercialschedules-application/src/main/java/org/dcsa/commercialschedules/controller/CommercialSchedulesController.java]: Added for point-to-point. Do we need for port-schedules? There isn't any limit parameter for port schedules.

**Review by douradofrazer — APPROVED** [review]

**Review by douradofrazer — APPROVED** [review]

**Review by douradofrazer — APPROVED** [review]

**Pull request merged**